portugues/colabora/comdiferencas/diff/corrigeepullrequest/corrigeepullrequest.md

```diff
@@ -0,0 +1,4 @@
+O responsável pelo repositório de origem verifica que outros novos 13 bugs apareceram depois das suas alterações e envia
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.